src/lib/registration/sw_mgmt.rb

```diff
@@ -60,9 +60,11 @@ class SwMgmt
 
     ZYPP_DIR = "/etc/zypp".freeze
 
-    FAKE_BASE_PRODUCT = { "name" => "SLES", "arch" => "x86_64", "version" => "12-0",
+    FAKE_BASE_PRODUCT = { "kind" => :product,
```

Comment:
I have change it back to a hash in order not to change the interface of the swmgmt class. 
Otherwise it would break a lot of other stuff in the modules which are using swmgmt.